src/hotspot/share/interpreter/oopMapCache.cpp

```diff
@@ -178,10 +175,18 @@ class VerifyClosure : public OffsetClosure {
 InterpreterOopMap::InterpreterOopMap() {
   initialize();
 #ifdef ASSERT
-  _resource_allocate_bit_mask = true;
+  _used = false;
 #endif
```

Comment:
Nit pre-existing: use of DEBUG_ONLY would be more consistent with later setting of `_used`.

Reply:
Fixed.